tasks/sep/006-0-sc-changes/SignFromJson.s.sol

```diff
@@ -0,0 +1,523 @@
+    function getRequiredThreshold(uint256 _numOwners) external view returns (uint256 threshold_);
+    function livenessGuard() external view returns (address livenessGuard_);
+    function livenessInterval() external view returns (uint256 livenessInterval_);
+    function minOwners() external view returns (uint256 minOwners_);
+    function safe() external view returns (address safe_);
+    function thresholdPercentage() external view returns (uint256 thresholdPercentage_);
+    function version() external view returns (string memory);
+}
+
+interface ILivenessGuardFetcher {
+    function lastLive(address) external view returns (uint256);
+    function safe() external view returns (address safe_);
+    function version() external view returns (string memory);
+}
+
+interface IDeputyGuardianModuleFetcher {
+    function deputyGuardian() external view returns (address deputyGuardian_);
+    function safe() external view returns (address safe_);
+    function superchainConfig() external view returns (address superchainConfig_);
+    function version() external view returns (string memory);
+}
+
+contract SignFromJson is OriginalSignFromJson {
+    using LibString for string;
+
+    // Chains for this task.
+    string constant l1ChainName = "sepolia";
+    string constant l2ChainName = "op";
+
+    // Safe contract for this task.
+    GnosisSafe securityCouncilSafe = GnosisSafe(payable(0xa87675ebb9501C7baE8570a431c108C1577478Fa));
+    GnosisSafe foundationSafe = GnosisSafe(payable(0xDEe57160aAfCF04c34C887B5962D0a69676d3C8B));
+
+    // Contracts we need to check, which are not in the superchain registry
+    IDeputyGuardianModuleFetcher deputyGuardianModule =
+        IDeputyGuardianModuleFetcher(0x2329EfD0bFc72Aa7849D9DFc2e131D83F4680d85);
+    ILivenessGuardFetcher livenessGuard = ILivenessGuardFetcher(0x54E8baCcC67fA3c6b3e9A94BAa4d70d1668f0820);
+    ILivenessModuleFetcher livenessModule = ILivenessModuleFetcher(0xefd77C23A8ACF13E194d30C6DF51F1C43B0f9932);
+
+    // Known EOAs to exclude from safety checks.
+    address constant l2OutputOracleProposer = 0x49277EE36A024120Ee218127354c4a3591dc90A9; // cast call $L2OO "PROPOSER()(address)"
+    address constant l2OutputOracleChallenger = 0xfd1D2e729aE8eEe2E146c033bf4400fE75284301; // In registry addresses.
+    address constant systemConfigOwner = 0xfd1D2e729aE8eEe2E146c033bf4400fE75284301; // In registry addresses.
+    address constant batchSenderAddress = 0x8F23BB38F531600e5d8FDDaAEC41F13FaB46E98c; // In registry genesis-system-configs
+    address constant p2pSequencerAddress = 0x57CACBB0d30b01eb2462e5dC940c161aff3230D3; // cast call $SystemConfig "unsafeBlockSigner()(address)"
+    address constant batchInboxAddress = 0xff00000000000000000000000000000011155420; // In registry yaml.
+
+    // The deployer address which should be removed as an owner on the security council safe.
+    address constant deployerAddress = 0x78339d822c23D943E4a2d4c3DD5408F66e6D662D;
```

Comment:
What's this part / why do we remove this owner?

Reply:
oh, that is outdated fixing locally.